File: 07/Python3/Governador.py
#!/usr/bin/python3
# coding: utf-8
"""Classe Governador """

from Politico import Politico

class Governador(Politico):
    """ Classe Senador """

    def __init__(self, nome, partido, estado):
        """ Construtor da classe Governador"""
        Politico.__init__(self)
        self.set_nome(nome)
        self.set_partido(partido)
        self.set_estado(estado)
        self.set_funcao("administrar bem o ICMS para o bem do estado.")

    def apresentacao(self):
        Politico.apresentacao(self)
        print ('sou Governador ')
        print ('Minha função é ' + self.get_funcao())
        print ('Fui eleito por ' + self.get_estado())
        print ('============================')

File: Projetos/Políticos/C++/sources/Governador.cpp
// Governador.cpp
#include <stdio.h>
#include <stdlib.h>
#include <iostream>
#include "../includes/Governador.h"
using namespace std;

Governador::Governador(string nome,string partido, string estado) {
    setNome(nome);
    setSalario(22000);
    setPartido(partido);
    setEstado(estado);
    setFuncao("administrar bem o ICMS para o bem do estado. ");
}
Governador::~Governador() {
    setNome("");
    setPartido("");
    setEstado("");
}
void Governador::apresentacao(){
    Politico::apresentacao();
    cout << "governador" << endl;
    cout << "Minha função é " << getFuncao();
    cout << getEstado() << endl;
    cout << "============================" << endl;
}
